projects/ancestor/ancestor.py

```diff
@@ -13,8 +12,33 @@ def add_edge(self, v1, v2):
         if v2 not in self.vertices:
             self.add_vertices(v2);
         self.vertices[v1].add(v2)
-    # Build graph
+
+
+def earliest_ancestor(ancestors, starting_node):
```

Comment:
Your solution code for earliest ancestor exercise currently fails the unit test.